test/units/testsuite-09.service

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+[Unit]
+Description=TEST-08-REBOOT
```

Comment:
TEST-09-REBOOT

Reply:
I fixed this locally and pushed to `main`.